src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -556,7 +556,7 @@ function BaseSelectionList<TItem extends ListItem>(
                     {/* If we are loading new options we will avoid showing any header message. This is mostly because one of the header messages says there are no options. */}
                     {/* This is misleading because we might be in the process of loading fresh options from the server. */}
                     {!isLoadingNewOptions && !!headerMessage && (
-                        <View style={headerMessageStyle ?? [styles.ph5, styles.pb5]}>
+                        <View style={headerMessageStyle ?? [styles.ph4, styles.pb5]}>
```

Comment:
Wait apologies I approved too soon. We want the padding on these pages to be 20px horizontal, not 16. Can we make sure that everything is wrapped within 20px of padding, including the search input?

Reply:
Thanks Brandon!